packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Overview of the mobile Identity Engine SDK
+---
+
+<div class="oie-embedded-sdk">
+
+Design the implementation of the sign-in flow for your mobile app by understanding the representation of the flow, and with examples of the common parts.
+
+<ApiLifecycle access="ie" /><br>
+
+## Introduction
+
+Integrate the Okta sign-in flow into your app in one of two ways: redirect the request to Okta which handles the flow in its own window, or handle each step in the flow by building the appropriate user interface elements, capturing the information, and updating Okta.
+
+This guide is an overview of building your own user interface. For information on the redirect model, see <StackSnippet snippet="redirectquickstart" inline />.
+
+## Sign-in flow
+
+Okta supports many ways of authenticating the identity of a user during the sign-in flow. An Okta org administrator creates _policies_, different mixes of authentication methods (also called factors), and assigns them to apps, people, groups, and more. Policies also configure how many authentication methods are required (multifactor), and which methods are optional. This results in a large number of possible combinations for the different factors and steps to complete the sign-in flow.
+
+The Android and iOS Identity Engine SDKs represent the sign-in flow as a state machine. You initialize the machine with the details of your Okta Application Integration, request the initial step in the flow, and cycle through responding to steps until either the user is signed in, cancels, or an error occurs.
```

Comment:
The phrase "is signed in" includes the notion of "success", otherwise they couldn't be signed in. What I did do was make plurality match:

> ...signs in, cancels, ...